packages/flutter/lib/src/material/tooltip_theme.dart

```diff
@@ -250,3 +280,24 @@ class TooltipTheme extends InheritedTheme {
   @override
   bool updateShouldNotify(TooltipTheme oldWidget) => data != oldWidget.data;
 }
+
+/// The method of interaction that will trigger a tooltip.
+/// Used in [Tooltip.triggerMode] and [TooltipThemeData.triggerMode]
+enum TooltipTriggerMode {
+  /// Tooltip will only be shown by calling `ensureTooltipVisible`
+  manual,
+
+  /// Tooltip will be shown after a long press.
+  ///
+  /// See Also:
+  ///
+  /// * [GestureDetector.onLongPress]
```

Comment:
```suggestion
  ///   * [GestureDetector.onLongPress] <-- same here and elsewhere, there should be context provided -->
```

Reply:
What kind of context are you talking about?